internal/winapi/cimfs.go

```diff
@@ -32,11 +32,17 @@ type CimFsFileMetadata struct {
 	EACount            uint32
 }
 
+type CimFsImagePath struct {
+	ImageDir  *uint16
+	ImageName *uint16
+}
+
 //sys CimMountImage(imagePath string, fsName string, flags uint32, volumeID *g) (hr error) = cimfs.CimMountImage?
 //sys CimDismountImage(volumeID *g) (hr error) = cimfs.CimDismountImage?
 
 //sys CimCreateImage(imagePath string, oldFSName *uint16, newFSName *uint16, cimFSHandle *FsHandle) (hr error) = cimfs.CimCreateImage?
-//sys CimCloseImage(cimFSHandle FsHandle) = cimfs.CimCloseImage?
+//sys CimCreateImage2(imagePath string, flags uint32, oldFSName *uint16, newFSName *uint16, cimFSHandle *FsHandle) (hr error) = cimfs.CimCreateImage2?
+//sys CimCloseImage(cimFSHandle FsHandle) = cimfs.CimCloseImage
```

Comment:
Was removing the `?` intentional?

Reply:
But the point of `?` is that it returns an error instead of panicking if the DLL/function cannot be loaded. So isn't it returning an error what we want?

Reply:
To me, it seems like we should decide how we want to treat a DLL import load failure: we can panic, or return an error. Then we should be consistent with that behavior (so either `?` everywhere, or nowhere).

If we add the `?` here, it won't be a "garbage" value, it will just be nil unless the import load failed.

Personally, my view is that returning an error is better, since if the node is messed up DLL load failures could happen and that makes sense to treat as an error case to me. But the more important thing is that we're consistent.

Reply:
I have updated this. 

Quick explanation for future reference: If the syscall definition in `internal/winapi/cimfs.go` is written as `//sys CimCloseImage(cimFSHandle FsHandle) = cimfs.CimCloseImage?` it generates the function that still returns an error in case of failure to load the DLL etc. but doesn't examine the return value of the syscall itself. However, if the syscall definition is written as `//sys CimCloseImage(cimFSHandle FsHandle) (hr error) = cimfs.CimCloseImage?` then it examines the return value of the syscall as well, which we don't want since this syscall doesn't return anything.